test/mocha/let.js

```diff
@@ -0,0 +1,30 @@
+var Uglify = require('../../');
+var assert = require("assert");
+
+describe("let", function() {
+    it("Should not produce `let` as a variable name in mangle", function(done) {
+        this.timeout(10000);
+
+        // Produce a lot of variables in a function and run it through mangle.
+        var s = '"use strict"; function foo() {';
+        for (var i = 0; i < 21000; ++i) {
```

Comment:
Because it takes too long to test. On my machine the test for 21,000 variables is 1 second, but on Travis CI machine it is over 2 seconds. 266K vars would take around 24s or so on Travis CI.